### PR TITLE
docker: lighten Dockerfile

### PR DESCRIPTION
**Pull request by sduthil**
Before: 637MB
After: 128MB

### CODE REVIEW

**Comment by fblackburn1** [comment]
Out of curiosity, do you know if this image is used somewhere?

**Comment by sduthil** [comment]
No, I don't think it is currently used.

File: Dockerfile [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
+COPY --from=compile-image /opt/venv /opt/venv
+
+COPY ./etc/wazo-dxtora /etc/wazo-dxtora
+
+RUN true \
+    && adduser --quiet --system --group --home /var/lib/wazo-dxtora wazo-dxtora \
+    && install -o wazo-dxtora -g wazo-dxtora /dev/null /var/log/wazo-dxtora.log \
+    && install -d -o wazo-dxtora -g wazo-dxtora /run/wazo-dxtora
+
+EXPOSE 9500
```
Comment (bloom1) [inline Dockerfile:26]: I'm not sure I understand why we need to expose this port

Comment (sduthil) [reply Dockerfile]: We don't, it's gone :) Thanks!

**Review by bloom1 — APPROVED** [review]

**Pull request merged**